security/docker/Dockerfile.node-agent-k8s

```diff
@@ -3,11 +3,12 @@ ARG BASE_DISTRIBUTION=default
 
 # The following section is used as base image if BASE_DISTRIBUTION=default
 FROM istionightly/base_debug as default
+RUN apt-get update && apt-get install -y --no-install-recommends ca-certificates && update-ca-certificates && apt-get clean  && rm -rf /var/lib/apt/lists/*
 
 # The following section is used as base image if BASE_DISTRIBUTION=distroless
 FROM gcr.io/distroless/static as distroless
 
 # This will build the final image based on either default or distroless from above
 FROM ${BASE_DISTRIBUTION}
-ADD node_agent_k8s /
+COPY node_agent_k8s /
```

Comment:
was the root cause of the need for chmdo this add vs copy?

We will need to keep that in mind in the future if so.

Reply:
COPY vs ADD is only an optimization.